### PR TITLE
packages, packages on DOM, New Package Form w/ connected button created

### PR DESCRIPTION
**Pull request by AndrewSpur73**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,67 @@
...
+    name: document.querySelector("#projectName").value,
+    description: document.querySelector("#exampleFormControlTextarea1").value,
+  };
+  packages.push(newPackageObj);
+  allPackages(packages);
+  form.reset();
+});
+};
+
+  const startApp = () => {
```
Comment (britnay268) [inline main.js:61]: Add navOnDom before allPackages(form)

File: main.js [hunk 2]
```diff
@@ -95,7 +107,62 @@ const profileOnDom = (array) => {
...
+      id: packages.length + 1,
+      name: document.querySelector("#projectName").value,
+      description: document.querySelector("#exampleFormControlTextarea1").value,
+    };
+    packages.push(newPackageObj);
+    allPackages(packages);
+    form.reset();
+  });
+};
+
```
Comment (britnay268) [inline main.js:158]: Add an evenListener for .nav on line160 and make sure to put e.preventDefault(); in the event Listener as it constantly reloads the page. Then add an if statement that checks if an id is present in the packages in the navbar. (There is not currently an id but I will make a comment to add an id) After you check if the id is present, it should call allpackages and packageForm. Should look something like this:
image

File: main.js [hunk 3]
```diff
@@ -22,17 +37,15 @@ const navOnDom = () => {
       <a class="nav-link" href="#">Projects</a></a>
     </li>
     <li class="nav-item">
-      <a class="nav-link" href="#">Packages</a>
+      <a class="nav-link" href="packages.html">Packages</a>
```
Comment (britnay268) [inline main.js:40]: add an id called nav-packages

File: main.js [hunk 4]
```diff
@@ -95,7 +107,62 @@ const profileOnDom = (array) => {
...
+      description: document.querySelector("#exampleFormControlTextarea1").value,
+    };
+    packages.push(newPackageObj);
+    allPackages(packages);
+    form.reset();
+  });
+};
+
+//On Start Function
+const startApp = () => {
```
Comment (britnay268) [inline main.js:160]: remove allPackages(packages) and packageForm()

**Review by britnay268 — APPROVED** [review]
Solid!

**Review by ursapictura — APPROVED** [review]

**Pull request merged**